app/scripts/modules/core/pipeline/config/stages/cloneServerGroup/aws/cloneServerGroupExecutionDetails.controller.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.stage.cloneServerGroup.aws.executionDetails.controller', [
+  require('../../../../../utils/lodash.js'),
+  require('angular-ui-router'),
+  require('../../../../../cluster/filter/clusterFilter.service.js'),
+  require('../../../../../delivery/details/executionDetailsSection.service.js'),
+  require('../../../../../delivery/details/executionDetailsSectionNav.directive.js'),
+  require('../../../../../navigation/urlBuilder.service.js'),
+])
+  .controller('awsCloneServerGroupExecutionDetailsCtrl', function ($scope, $stateParams, executionDetailsSectionService, $timeout, urlBuilderService, clusterFilterService) {
+
+    $scope.configSections = ['cloneServerGroupConfig', 'taskStatus'];
+
+    function initialize() {
+      executionDetailsSectionService.synchronizeSection($scope.configSections);
+      $scope.detailsSection = $stateParams.details;
+
+      // When this is called from a stateChangeSuccess event, the stage in the scope is not updated in this digest cycle
+      // so we need to wait until the next cycle to update the deployed artifacts
+      $timeout(() => {
+        let context = $scope.stage.context || {},
+          results = [];
+
+        function addDeployedArtifacts(key) {
+          let deployedArtifacts = _.find(resultObjects, key);
+          if (deployedArtifacts) {
+            _.forEach(deployedArtifacts[key], (serverGroupNameAndRegion) => {
+              if (serverGroupNameAndRegion.includes(':')) {
+                let [region, serverGroupName] = serverGroupNameAndRegion.split(':');
+                let result = {
+                  type: 'serverGroups',
+                  application: context.application,
+                  serverGroup: serverGroupName,
+                  account: context.credentials,
+                  region: region,
+                  provider: context.providerType || context.cloudProvider || 'aws',
```

Comment:
since this stage is the AWS implementation, you can just set this to `aws`, I think